moveit_simple/src/moveit_simple.cpp

```diff
@@ -62,7 +60,15 @@ Robot::Robot(const ros::NodeHandle & nh, const std::string &robot_description,
   return;
 }
 
+Robot::Robot(const ros::NodeHandle &nh, const std::string &robot_description,
+  const std::string &group_name, const std::string &ik_base_frame,
+  const std::string &ik_tip_frame) : Robot(nh, robot_description, group_name)
+{ 
+  ik_base_frame_ = ik_base_frame;
```

Comment:
Why aren't `ik_base_frame` and `ik_tip_frame` set in the default constructor as the first/last links in the moveit chain.  Not setting class values to appropriate values is a recipe for bugs.  Other developers may not understand that they are conditionally set.  Also, if the values are appropriate no matter how the object is constructed then the class methods can operate on them without checking a condition variable.  This will make your methods more straight-forward.

Reply:
Done.